diff --git a/lightkeeper/src/main/java/lightkeeper/LightKeeperPlugin.java b/lightkeeper/src/main/java/lightkeeper/LightKeeperPlugin.java
index 3979ad1..3882524 100644
--- a/lightkeeper/src/main/java/lightkeeper/LightKeeperPlugin.java
+++ b/lightkeeper/src/main/java/lightkeeper/LightKeeperPlugin.java
@@ -15,6 +15,7 @@
  */
 package lightkeeper;
 
+import ghidra.app.CorePluginPackage;
 import ghidra.app.plugin.PluginCategoryNames;
 import ghidra.app.plugin.ProgramPlugin;
 import ghidra.framework.plugintool.PluginInfo;
@@ -32,7 +33,7 @@ import lightkeeper.model.table.CoverageTableModel;
 import lightkeeper.view.LightKeeperProvider;
 
 //@formatter:off
-@PluginInfo(status = PluginStatus.STABLE, packageName = "light keeper", category = PluginCategoryNames.MISC, shortDescription = "Plugin for visualization of DynamoRio coverage data.", description = "Plugin for visualization of DynamoRio coverage data.")
+@PluginInfo(status = PluginStatus.STABLE, packageName = CorePluginPackage.NAME, category = PluginCategoryNames.MISC, shortDescription = "Plugin for visualization of DynamoRio coverage data.", description = "Plugin for visualization of DynamoRio coverage data.")
 //@formatter:on
 public class LightKeeperPlugin extends ProgramPlugin {
 	protected CoverageModel coverageModel;
